include/dsn/dist/replication/replication_ddl_client.h

```diff
@@ -274,10 +278,31 @@ class replication_ddl_client
         }
     }
 
+    /// Send request to single replica server synchronously.
+    template <typename TRpcHolder, typename TResponse = typename TRpcHolder::response_type>
+    error_code call_rpc_async(dsn::rpc_address address,
+                              TRpcHolder rpc,
+                              /*out*/ TResponse &resp)
+    {
+        dsn::task_tracker tracker;
+        error_code err = ERR_UNKNOWN;
+        rpc.call(address, &tracker, [&err, &resp, rpc](error_code code) mutable {
+            err = code;
+            if (err == dsn::ERR_OK) {
+                resp = rpc.response();
+            } else {
+                err = code;
+            }
+        });
+        tracker.wait_outstanding_tasks();
+        return err;
+    }
+
```

Comment:
So you command would support send to one node meanwhile? can you use `call_rpcs_async` directly which can send one or more node meanwhile

Reply:
I have tried to code like this, but it is very complicated.
I think it's ok to add an interface to support this function.

Reply:
I agree with @Shuo-Jia comment, you can use function `call_rpcs_sync`.